w3c.json

```diff
@@ -1,5 +1,5 @@
 {
   "group": [96877, 125519],
-  "contacts": ["tidoust", "Kangz", "grorg"],
+  "contacts": ["tidoust", "Kangz", "grorg", "kdashg"],
```

Comment:
nit: can you remove "grorg"? Dean hasn't been chair for quite some time now.

Reply:
@Kangz done 😿